test/deployment-proposal.test.ts

```diff
@@ -659,8 +659,7 @@ describe("proposal", function () {
         gasPrice: 545019933,
         gasLimit,
       };
-      const signed = await ambExecutor.signTransaction(tx);
-      await hre.ethers.provider.sendTransaction(signed);
+      await ambExecutor.sendTransaction(tx);
```

Comment:
I noticed that this test failed when adding new tests in completely unrelated files.
Probably, signing a transaction and then executing it somehow breaks the nonce management in Hardhat tests.

Reply:
Interesting. Good find.